jans-cedarling/bindings/cedarling_python/PYTHON_TYPES.md

```
@@ -65,6 +65,83 @@ Attributes
+
+Example
+-------
+```python
+req = BatchAuthorizeMultiIssuerRequest(
+    tokens=[TokenInput(mapping="Jans::Access_Token", payload="eyJ...")],
+    items=[BatchItem(resource=res1, action="Read", context={}), ...],
+)
+```
+---
+
+BatchAuthorizeMultiIssuerResponse
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)